src/pages/library/DataBrowserDetails.js

```diff
@@ -1,57 +1,161 @@
+  content: { padding: 20, marginTop: 15 },
+  headers: { margin: '20px 0 0' },
+  attributesColumn: { width: '22%', marginRight: 20, marginTop: 30 },
+  access: {
+    open: colors.success(1.5),
+    controlled: colors.accent()
+  }
 }
 
 const getSnapshot = id => new Promise(resolve => setTimeout(() => {
-  resolve({
-    namespace: 'harry-potter',
-    name: 'Harry Potter',
-    created: '2020-01-07T18:25:28.340Z',
-    lastUpdated: '2020-01-07T18:25:28.340Z',
-    tags: {
-      itemsType: 'AttributeValue',
-      items: ['1000 Genomes', 'CMG', 'Open Access', 'Exome']
-    },
-    description: 'The boy who lived',
-    lowerName: 'harry potter',
-    lowerDescription: 'lower description',
-    project: {
-      id: '112',
-      name: 'CDC'
-    },
-    subjects: 10,
-    dataType: 'RNA Seq',
-    keepCollapsed: true,
-    locked: false,
-    files: 15,
-    fileSize: 4432
+  fetch('hca-sample.json').then(response => {
+    response.json().then(tempData => {
+      const dataMap = _.keyBy('dct:identifier', tempData.data)
+      resolve(dataMap[id])
+    })
   })
-}, 2000))
+}, 1000))
 
 const MainContent = ({ snapshot }) => {
   return div({ style: { ...styles.content, width: '100%', marginTop: 0 } }, [
-    h1({ style: { marginTop: 0 } }, [snapshot.name]),
-    div({ style: { whiteSpace: 'pre-wrap' } }, [JSON.stringify(snapshot, null, 2)])
+    h1([_.get('dct:title', snapshot)]),
+    div([_.get('dct:description', snapshot)]),
+    h2({ className: 'sr-only' }, ['Snapshot Sources']),
+    div({ style: { display: 'flex', width: '100%', flexWrap: 'wrap' } }, [
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Data release policy']),
+        div([_.get('releasePolicy', snapshot)])
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Last Updated']),
+        div([Utils.makeStandardDate(_.get('dct:modified', snapshot))])
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Version']),
+        div([_.get('', snapshot)])
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Region']),
+        _.map(
+          storage => div({ key: `region-table-${storage.region}` }, [storage.region]),
+          _.uniqBy('region', snapshot.storage)
+        )
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Contact'])
+        // h(Link, { href: `mailto:fakeemail@fake.org` }, ['fakeemail@fake.org'])
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Data curator'])
+        // div(['Will add later, after data structure is added'])
+      ]),
+      div({ style: styles.attributesColumn }, [
+        h3({ style: styles.headers }, ['Contributors']),
+        _.map(contributor => div({ key: `contributor-list_${contributor}}` }, [contributor]), _.get('contributors', snapshot))
```

Comment:
Could we not just join the array with newlines and render the resultant string in a div with linebreaks preserved, instead of mapping?

Reply:
We could, I'm kind of future-proofing a little bit here, since I'm not sure what the contributors will look like, or if they will a section to render their like contact information! This is dependent on data that doesnt exist yet!